src/events/message_create.ts

```diff
@@ -49,7 +51,12 @@ export async function evergreenIssueWorkflow(message: Message) {
   ].filter(Boolean);
 
   // TODO(rayhanadev): generate title using groq
-  const title = `Evergreen request from @${people[message.author.id] ?? message.author.tag} in #${message.channel.name}`;
+  let title = `Evergreen request from @${people[message.author.id] ?? message.author.tag} in #${message.channel.name}`;
+
+  if (message.content.match(/evergreen it\s?/i)) {
```

Comment:
This doesn't seem to only match the start of the string, it matches anywhere in the message body. Might want to fix this to prevent accidental uses.

<img width="831" alt="image" src="https://github.com/user-attachments/assets/cde3f4b6-7884-426e-a144-c9a3439e72cd" />

Reply:
Resolved